locales/en/kafka.json

```diff
@@ -122,6 +122,24 @@
   "partitions": "Partitions",
   "partitions_value": "up to 500",
   "select_filter_category": "Select filter category",
+  "settings": {
+    "connection_re_authentication_label": "Connection re-authentication.",
+    "Connection_re_authentication_states": {
+      "turning_off": "Turning off...",
+      "turning_on": "Turning on...",
+      "turnoff": "Off",
+      "turnon": "On"
+    },
+    "connection_re_authentication_switch": "Connection re-authentication.",
+    "connection_re_authentication_title": "Connection re-authentication:",
```

Comment:
You have three instances of "Connection re-authentication" defined here. Are all of these necessary? Two of these are being used for aria-labels. In these cases, the aria-label should be a match for the visible text, which means we should be able to use a single string going forward. 

Something to note is that this pattern should eventually become a shared component that is reused, either here for more settings or for other services. Having a simpler way to specify a single string that's used in all the places would help avoid issues with using this pattern in the future.

cc'ing @riccardo-forina in case I'm saying anything that is way off here.

Reply:
Also, the first two instances of this text string include a period (.) at the end but typically we don't include periods in labels, only for complete sentences.

The other question I have is whether we should include the colon (:) in the third instance. That's not necessary for translation, and I wonder if it would be better to move that out of the text file and bake it into the UI pattern instead.

Reply:
@jgiardino looking into the doc https://docs.google.com/presentation/d/1cq6TnLYjntOZZYmXoAgCO9YuTlPRM8-VzqxbBgRi5EI/edit#slide=id.g132fbec33c0_0_23 `Connection re-authentication:` seems like a single string so I have included colon(:) in the translation file

Reply:
I think this question comes down to how do we abstract this UI into a reusable pattern and also make this UI easy to maintain. 

Displaying the colon (:) is part of the pattern, and also isn't something that needs to be translated. To reduce the risk of inconsistency, ideally we would separate what the dev needs to specify when reusing this pattern, and not require that they provide the colon as part of the string. 

I would also reduce the number of instances in the translation file that include the same exact string. This requires more effort if changes are needed, but also increases the risk that changes will be made on one string, but not the other.